src/components/CurrentUser/CurrentUser.tsx

```diff
@@ -1,12 +1,59 @@
 import React from 'react';
+import { loadUsers } from '../../api';
 import './CurrentUser.scss';
 
-export const CurrentUser: React.FC = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+type Props = {
+  selectedUserId: number;
+  clear: () => void;
+};
 
-    <h3 className="CurrentUser__name">Ervin Howell</h3>
-    <p className="CurrentUser__email">Shanna@melissa.tv</p>
-    <p className="CurrentUser__phone">010-692-6593 x09125</p>
-  </div>
-);
+type State = {
+  selectedUser: User | null
+};
+
+export class CurrentUser extends React.Component<Props, State> {
+  state: State = {
+    selectedUser: null,
+  };
+
+  async componentDidMount() {
+    await this.loadUsers();
+  }
+
+  componentDidUpdate(prevProps: Props) {
+    if (prevProps.selectedUserId !== this.props.selectedUserId) {
+      this.loadUsers();
+    }
+  }
+
+  async loadUsers() {
```

Comment:
это один пользователь, а не множество
```suggestion
  async reloadUser() {
```